daemon/bond_mon.c

```diff
@@ -375,6 +376,7 @@ bm(
     static MTC_BOND_STATUS  bond_status = BOND_STATUS_NOERR;
     PCOM_DATA_BM            pbm;
 
+    log_message(MTC_LOG_INFO, "BM: thread ID: %ld.\n", syscall(SYS_gettid));
```

Comment:
It might be nicer to create a `gettid()` wrapper function than using a raw `syscall()` everywhere.